File: app/models/Module.py
class Module:
    id: int
    physical_id: str
    referencePoint: bool
    xCoordinate: float
    yCoordinate: float

    def __init__(self, id: int, physical_id: str, referencePoint: bool, xCoordinate: float, yCoordinate: float):
        self.id = id
        self.physical_id: physical_id
        self.referencePoint = referencePoint
        self.xCoordinate = xCoordinate
        self.yCoordinate = yCoordinate
    
    def __init__(self, physical_id: str, referencePoint: bool, xCoordinate: float, yCoordinate: float):
        self.physical_id: physical_id
        self.referencePoint = referencePoint
        self.xCoordinate = xCoordinate
        self.yCoordinate = yCoordinate

File: rubble-rescue-ui/src/components/mapItems/Module.js
import React from "react";
import classes from './Module.module.css';

const Module = (props) => {
    // let coords = props.module['coordinates'].split(", ");
    let left = `${Math.floor(parseFloat(props.module['xCoordinate'])) + 40}%`;
    let top = `${Math.floor(parseFloat(props.module['yCoordinate'])) - 10}%`;
    
    const style = {"left": left, "top": top}

    return ( <img src={require('../../assets/module.png')} className={classes.Image} style={style} alt="Module" /> );
}

export default Module;